api-reference/10 UI Components/dxDateRangeBox/1 Configuration/disableOutOfRangeSelection.md

```diff
@@ -5,7 +5,10 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether the UI component disables dates selection before the start date and after the end date.
```

Comment:
It can be either "selection of dates" (plural) or "date selection" (singular, because used as a modifier). 

```suggestion
Specifies whether the UI component disables date selection before the start date and after the end date.
```